src/olympia/bandwagon/views.py

```diff
@@ -576,11 +576,12 @@ class CollectionViewSet(ModelViewSet):
             # (it's community-managed) and change it's addons, but can't delete
             # or edit it's details.
             AllOf(AllowCollectionContributor,
-                  PreventActionPermission(['create', 'list', 'update',
-                                           'destroy', 'partial_update'])),
-            # Admins can do everything except create.
-            AllOf(GroupPermission(amo.permissions.COLLECTIONS_EDIT),
-                  PreventActionPermission('create')),
+                  PreventActionPermission(('create', 'list', 'update',
+                                           'destroy', 'partial_update'))),
+            # Content curators can modify existing mozilla collections as they
+            # see fit.
+            AllOf(AllowContentCurators,
+                  PreventActionPermission(('create', 'list'))),
```

Comment:
I think we should block `destroy` too - I can't think of any good reason why a content curator should be deleting Mozilla branded collections via frontend.  We could give them `list` though, as it's harmless for people with `ADMIN_CURATION` (though admittedly may also be useless if you can't access the list from frontend)

Reply:
`list` is an issue, because it would allow anyone to list, with the way permission checks are done when listing. Since it was useless, I didn't add it...

Agree about destroy.

Reply:
https://github.com/mozilla/addons-server/pull/8651/commits/a7bc753adb77b94586f01bfe62f02a03ccc7efd6 prevents destroy. I tried allowing list but since the permission never receives the user, it can't easily be used to restrict curators to only list mozilla-owned collections, so I left it out.